src/components/Images/Images.js

```diff
@@ -46,7 +40,6 @@ export const CloudinaryImage = ({
   width,
   height,
   crop,
-  lazyLoad,
```

Comment:
we should no longer use `lazyload` flag for `Cloudinary` image. I will have a follow-up PR in CMS to remove the use of this flag.